addon/background.js

```diff
@@ -0,0 +1,206 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public

```

Comment:
This is where everything happens for the webextension.  Most of what is in there was pulled from `src/webextension/lib/{browserAction, storage}` and `src/lib/pings`.